src/view/com/posts/Feed.tsx

```diff
@@ -294,7 +294,7 @@ let Feed = ({
         scrollEventThrottle={scrollEventThrottle}
         indicatorStyle={theme.colorScheme === 'dark' ? 'white' : 'black'}
         onEndReached={onEndReached}
-        onEndReachedThreshold={2}
+        onEndReachedThreshold={3} // number of posts left to trigger load more
```

Comment:
Do you want this in this PR?

Reply:
Probably not. We can test it more later.